pkg/locking/locking_test.go

```diff
@@ -32,6 +32,16 @@ func TestLockingTwiceThrowsError(t *testing.T) {
 	}
 	state2, err2 := pl2.Lock()
 	assert.False(t, state2)
-	// No error because the lock was not aquired
+	// No error because the lock was not acquired
 	assert.NoError(t, err2)
 }
+
+func TestGetLock(t *testing.T) {
+	lock, err := GetLock()
+	if err != nil {
+		print(err)
+
+	}
+	print(lock)
```

Comment:
NIT: should we remove these prints or keep them ?

Reply:
added todo to implement this test properly later